tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -122,6 +123,8 @@ export class ScalarCardComponent {
   @Input() showFullSize!: boolean;
   @Input() isPinned!: boolean;
 
+  @Input() newXAxisType!: ScaleType;
```

Comment:
newXAxisType => xScaleType?  (XAxisType seems to be something different than ScaleType in your data model. I don't really understand what the "new" is supposed to mean but I guess it means "ScaleType is the new XAxisType")